scripts/extrinsic_check.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+
+# This script:
+# - Checks for changes in transaction version in runtime/src/lib.rs
+# - Downloads latest release binary from gluwa/creditcoin (RELEASE_BIN)
+# - Compiles and build a binary from the current branch (HEAD_BIN)
+# - Runs the two nodes
+
+set -e
+
+HEAD_BIN=./target/release/creditcoin-node
+HEAD_WS=ws://localhost:9944
+RELEASE_WS=ws://localhost:9945
+
+runtimes=(
+  "creditcoin-node"
+)
+
+# First we fetch the latest released binary
+latest_release() {
+  curl -s "https://api.github.com/repos/$1/releases/latest" | jq -r '.tag_name'
+}
+
+latest_release=$(latest_release 'gluwa/creditcoin')
+RELEASE_BIN="./creditcoin-node"
+echo "[+] Fetching binary for Creditcoin version $latest_release"
+curl -L "https://github.com/gluwa/creditcoin/releases/download/$latest_release/creditcoin-v${latest_release}-x86_64-unknown-linux-gnu.tar.gz"  --output creditcoin.tar.gz && tar -xzf creditcoin.tar.gz || exit 1
+chmod +x "$RELEASE_BIN"
+git fetch --depth="${GIT_DEPTH:-100}" origin 'refs/tags/*:refs/tags/*'
+
+
+for RUNTIME in "${runtimes[@]}"; do
+  echo "[+] Checking runtime: ${RUNTIME}"
+
+  release_transaction_version=$(
+    git show "tags/$latest_release:runtime/src/lib.rs" | \
+      grep 'transaction_version'
+  )
+
+  current_transaction_version=$(
+    grep 'transaction_version' "./runtime/src/lib.rs"
+  )
+
+  echo "[+] Release: ${release_transaction_version}"
+  echo "[+] Ours: ${current_transaction_version}"
+
+  if [ ! "$release_transaction_version" = "$current_transaction_version" ]; then
+    echo "[+] Transaction version for ${RUNTIME} has been bumped since last release."
+    exit 0
+  fi
+
+  # Start running the nodes in the background
+  $HEAD_BIN --chain=./chainspecs/qaSpec.json --tmp &
+  $RELEASE_BIN --chain=./chainspecs/qaSpec.json --ws-port 9945 --tmp &
```

Comment:
```suggestion
  $HEAD_BIN --chain=local --tmp &
  $RELEASE_BIN --chain=local --ws-port 9945 --tmp &
```

The QA chainspec will force the nodes to run the wasm runtime (which will always be the same version) and  give false negaties.

The `local` chain specification will run the node's "local" runtime version, which should give us the results we want.